pkg/oc/cli/cmd/rsync/copyrsync.go

```diff
@@ -111,6 +110,21 @@ func (r *rsyncStrategy) Validate() error {
 	return nil
 }
 
+// escapeRsyncCommand wraps each element of the command in double quotes
+// if it contains any of the following: single quote, double quote, backslash, or space
+// It also replaces every pre-existing quoting character in the element with a pair.
+// Example: " -> ""
```

Comment:
Now it only replaces double-quotes, not other quoting characters, so this comment is misleading.

Is this rsync feature well known? It might be worth adding an explanation (e.g., "When it adds double quotes, it also doubles every pre-existing double-quote; rsync interprets a doubled double-quote within a double-quoted string as a single double-quote."), or possibly changing the name to `rsyncEscapeCommand` to emphasize that this function performs rsync-style escaping (as opposed to shell-style escaping of an rsync command).